packages/block-library/src/embed/edit.js

```diff
@@ -177,7 +177,8 @@ const EmbedEdit = ( props ) => {
 		}
 	}, [ preview, isEditingURL ] );
 
-	const blockProps = useBlockProps();
+	// Shows responsive embeds correctly by adding the `wp-embed-responsive` class.
+	const blockProps = useBlockProps( { className: 'wp-embed-responsive' } );
```

Comment:
The fix makes sense to me, but if we always want it to be responsive, then what's the point of this class? Could we remove it entirely? @ellatrix WDYT?

Reply:
Sounds good to me. I don't know what this class is for, I don't know much about the embed block.

Reply:
The aspect ratio css needs the class in:
https://github.com/WordPress/gutenberg/blob/b72adf8aaa24afd1b1777c641806ad738c523486/packages/block-library/src/embed/style.scss#L42-L89

Removing the classname from backend would require to duplicate the aspect ratio css to `editor.scss` and remove the required `wp-embed-responsive`. Don't think we can change `style.scss` because responsive embeds are opt-in for themes.

Reply:
But we are hardcoding the class here? How do we make it opt-in for themes in the editor?